luajit.krun

```diff
@@ -0,0 +1,96 @@
+import os
+from krun.vm_defs import (LuaVMDef)
+from krun import EntryPoint
+from krun.util import fatal
+
+# Who to mail
+MAIL_TO = []
+
+# Maximum number of error emails to send per-run
+#MAX_MAILS = 2
+
+DIR = os.getcwd()
+
+HEAP_LIMIT = 2097152  # KiB
+STACK_LIMIT = 8192  # KiB
+
+# Variant name -> EntryPoint
+VARIANTS = {
+    "default-lua": EntryPoint("bench.lua"),
+}
+
+ITERATIONS_ALL_VMS = 5  # Small number for testing.
+
+VMS = {
+    'Normal': {
+        'vm_def': LuaVMDef("builds/luajit"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+    'GC64': {
+        'vm_def': LuaVMDef("builds/luajit_gc64"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+    'NoJIT': {
+        'vm_def': LuaVMDef("builds/luajit_nojit"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+    'DualNum': {
+        'vm_def': LuaVMDef("builds/luajit_dualnum"),
+        'variants': ['default-lua'],
+        'n_iterations': ITERATIONS_ALL_VMS,
+    },
+}
+
+BENCHMARKS = {
+    "binarytrees" : 1,
+    "nbody" : 1,
+    "fasta": 1,
+    "richards": 1,
+    "spectralnorm": 1,
+    "fannkuch_redux": 1,
+    "md5": 1,
+    "series": 1,
+    "luacheck_parser": 1,
+    "luacheck": 1,
+    "capnproto_encode": 1,
+    "capnproto_decode": 1,
+    "jsonlua_encode": 1,
+    "jsonlua_decode": 1,
+}
+
+# list of "bench:vm:variant"
```

Comment:
I'd recommend adding commented one-line-per-vm wildcard lines here so you can quickly disable VMs.

Reply:
Also, add spaces `SKIP = [`. This was wrong in the Krun repo too! Oops.

Reply:
fixed